### DISCUSSION TITLE
Back-port Auto LODs to 3.6

### DISCUSSION THREAD

**Discussion by ClintochX** [godotengine/godot-proposals#5929] [3D]
What would be the most beneficial thing that'll be Considered amazing and useful for Godot 3.6? That'll definitely be auto LODs (Level of details) support

It'll be a very nice to have feature and I really need it for the game I'm currently developing using my Potato PC

**Comment by Calinou** [comment]
Automatic mesh LOD requires many low-level changes, both on the import side and the renderer side. To minimize the risk of regressions, Godot 3.6 is not planned to receive large sweeping changes of this kind. It also requires changing the mesh resource format in a way that may require breaking compatibility.

That said, there's an open pull request to reimplement visibility ranges (manual HLOD) in `3.x`: https://github.com/godotengine/godot/pull/53778
In the meantime, you can use the [godot-lod](https://github.com/godot-extended-libraries/godot-lod) add-on.
